modules/osdk-run-proxy.adoc

```diff
@@ -0,0 +1,132 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/golang/osdk-golang-tutorial.adoc
+// * operators/operator_sdk/ansible/osdk-ansible-tutorial.adoc
+// * operators/operator_sdk/helm/osdk-helm-tutorial.adoc
+
+ifeval::["{context}" == "osdk-golang-tutorial"]
+:golang:
+endif::[]
+ifeval::["{context}" == "osdk-ansible-tutorial"]
+:ansible:
+endif::[]
+ifeval::["{context}" == "osdk-helm-tutorial"]
+:helm:
+endif::[]
+
+[id="osdk-run-proxy_{context}"]
+= Enabling proxy support
+
+To support proxied clusters, your Operator must inspect the environment for the following standard proxy variables and pass the values to Operands:
+
+* `HTTP_PROXY`
+* `HTTPS_PROXY`
+* `NO_PROXY`
+
+.Prerequisites
+* A cluster with cluster-wide egress proxy enabled.
+
+[NOTE]
+====
+This tutorial uses `HTTP_PROXY` as an example environment variable.
```

Comment:
Perhaps this should go either before the Prereqs or possibly as a callout note in step 2 where it is actually used? Seems odd to be located in the Prereqs.